frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -103,7 +102,7 @@ const MakerForm = ({
 
   useEffect(() => {
     updateCoordinatorInfo();
-  }, [maker.coordinator]);
+  }, [maker.coordinator, coordinatorUpdatedAt]);
```

Comment:
With this change `Order current rate:` should appear again bellow the create button